### PR TITLE
[GraphOptimizer] Combine Rescale down with Relu node

### PR DESCRIPTION
**Pull request by jfix71**
*Description*: Add optimization to merge ReluNode(Rescale(in)) -> ReluNode(in). Similar to others that exist for the Arithmetic nodes.

*Testing*: Added unit test.

*Documentation*: N/A

### CODE REVIEW

**Comment by rdzhabarov** [comment]
Btw, what network has this pattern? Any public one?

File: tests/unittests/graphOptzTest.cpp [hunk 1]
```diff
@@ -1040,16 +1052,25 @@ TEST_F(GraphOptz, mergeRescaleWithArithmeticNode) {
...
-  unsigned count = 0;
-  for (auto &n : F->getNodes()) {
-    if (n.getKind() == kind) {
-      count++;
-    }
-  }
+/// Check that Relu can be merged with Rescale.
+TEST_F(GraphOptz, mergeRescaleWithRelu) {
+  Variable *input = mod_.createVariable(ElemKind::Int8QTy, {4, 10}, 0.5, 11,
+                                        "input", VisibilityKind::Public, true);
```
[the remark this answers is not in the document]
Comment (jfix71) [reply tests/unittests/graphOptzTest.cpp]: I have no idea :) fixed.

File: tests/unittests/graphOptzTest.cpp [hunk 2]
```diff
@@ -1040,16 +1052,25 @@ TEST_F(GraphOptz, mergeRescaleWithArithmeticNode) {
...
 
-  return count;
+  auto *rescale1 = F_->createRescaleQuantized(
+      "rescale", input, mod_.uniqueType(ElemKind::Int8QTy, {4, 10}, 0.4, 11));
+  auto *relu = F_->createRELU("add", rescale1);
```
[the remark this answers is not in the document]
Comment (jfix71) [reply tests/unittests/graphOptzTest.cpp]: Done

**Comment by jfix71** [comment]
I found it when running a quantized resnet when `shouldLower()` is false for Relu.

**Pull request merged**